### ISSUE TITLE
Error in the Installation in windows 8

### ISSUE THREAD

**Issue by chahmadbilal** [KimiNewt/pyshark#9]
Below Error is appearing which is not letting this module install on Windows 8. Please help.

Downloading https://pypi.python.org/packages/source/l/lxml/lxml-3.3.1.tar.gz#md5
=09401c787d9b2660b0d76c4c6baec0f3
Processing lxml-3.3.1.tar.gz
Writing c:\users\user\appdata\local\temp\easy_install-aik18s\lxml-3.3.1\setup.cfg
Running lxml-3.3.1\setup.py -q bdist_egg --dist-dir c:\users\user\appdata\local\
temp\easy_install-aik18s\lxml-3.3.1\egg-dist-tmp-oly1w6
Building lxml version 3.3.1.
Building without Cython.
ERROR: b"'xslt-config' is not recognized as an internal or external command,\r\n
operable program or batch file.\r\n"
*\* make sure the development packages of libxml2 and libxslt are installed **

Using build configuration of libxslt
C:\Python33\lib\distutils\dist.py:258: UserWarning: Unknown distribution option:
 'extra_require'
  warnings.warn(msg)
C:\Python33\lib\distutils\dist.py:258: UserWarning: Unknown distribution option:
 'bugtrack_url'
  warnings.warn(msg)
error: Setup script exited with error: Unable to find vcvarsall.bat

C:\Users\USER\Downloads\pyshark-master\src>

**Comment by trevorKirkby** [comment]
Yes... I successfully got and installed the wheel of lxml from this site (I have used that site a ton for dependencies)
Thing is, there seems to be another dependency.

"Processing dependencies for pyshark
Searching for trollius==1.0.4"

So I installed trollius because most packages provide wheels now: https://pypi.python.org/pypi/trollius

It still throws the error "Unable to find vcvarsall.bat" however. I am guessing this is because it wants version 1.0.4 but I got version 2.0
Is there a good solution to this?
